js/msp.js

```diff
@@ -406,11 +408,7 @@ var MSP = {
                     RC_tuning.RC_YAW_EXPO = parseFloat((data.getUint8(offset++) / 100).toFixed(2));
                     if (semver.gte(CONFIG.apiVersion, "1.20.0")) {
                         RC_tuning.rcYawRate = parseFloat((data.getUint8(offset++) / 100).toFixed(2));
-                    } else {
-                        RC_tuning.rcYawRate = 0;
```

Comment:
Why are you removing this? If this is not reset, the value will stay if the user switches from a craft with 3.0.0 to one with 2.9.x without restarting the configurator.